findbugs-exclude.xml

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<FindBugsFilter>
-	<Match>
-		<Class name="org.jenkinsci.plugins.xunit.XUnitBuilder" />
-		<Bug pattern="EI_EXPOSE_REP" />
-		<!-- getter are needed only for Snippet Generator, are not APIs -->
-	</Match>
-	<Match>
-		<Class name="org.jenkinsci.plugins.xunit.XUnitPublisher" />
-		<Bug pattern="EI_EXPOSE_REP" />
-		<!-- getter are needed only for Snippet Generator, are not APIs -->
-	</Match>
```

Comment:
Moved to a standard location in the latest plugin parent POM: `src/spotbugs/excludesFilter.xml`. Also the `EI_EXPOSE_REP` check is globally disabled in the latest plugin parent POM, so those lines are removed.